RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -649,6 +649,111 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 			attackMinor = useItem($item[beehive], false);
 		}
 		break;
+
+	case $class[Pig Skinner]:
+	case $class[Cheese Wizard]:
+	case $class[Jazz Agent]:
+		attackMinor = "attack with weapon";
+		if(canUse($skill[Ball Throw], true) && (enemy.physical_resistance < 80))
+		{
+			attackMajor = useSkill($skill[Ball Throw], true);
+			attackMinor = useSkill($skill[Ball Throw], true);
+			costMajor = mp_cost($skill[Ball Throw]);
+			costMinor = mp_cost($skill[Ball Throw]);
+		}
+		if(canUse($skill[Hot Foot], true) && (enemy.defense_element != $element[hot]) && !enemyCanBlocksSkills())
+		{
+			attackMajor = useSkill($skill[Hot Foot], true);
+			attackMinor = useSkill($skill[Hot Foot], true);
+			costMajor = mp_cost($skill[Hot Foot]);
+			costMinor = mp_cost($skill[Hot Foot]);
+		}
+		if(canUse($skill[Stop Hitting Yourself], true) && (enemy.physical_resistance < 80))
+		{
+			attackMajor = useSkill($skill[Stop Hitting Yourself], true);
+			attackMinor = useSkill($skill[Stop Hitting Yourself], true);
+			costMajor = mp_cost($skill[Stop Hitting Yourself]);
+			costMinor = mp_cost($skill[Stop Hitting Yourself]);
+		}
+		if((my_hp() / 0.7 < my_maxhp()) && canUse($skill[Second Wind], true))
```

Comment:
In the event there is a large hit before we can cast Second Wind, I wanted to get it cast asap and I figured that at 70% health it would be quick enough. I am indifferent to the actual percentage though, tbh, so I reduced it down to 50%.
It is much more efficient than healing out of combat because PS has really high HP. Before I added this change, I was finishing runs at 0 meat. After this change, I was finishing them at 50k+ meat.